shared/src/main/scala/uk/gov/homeoffice/drt/ports/PaxTypes.scala

```diff
@@ -10,12 +10,16 @@ sealed trait PaxType {
   def cleanName: String = getClass.getSimpleName.dropRight(1)
 }
 
+sealed trait GbrPaxType extends PaxType
+
 sealed trait EeaPaxType extends PaxType
 
 sealed trait NonEeaPaxType extends PaxType
 
 object PaxType {
   def apply(paxTypeString: String): PaxType = paxTypeString match {
+    case "GBRNational$" => GBRNational
```

Comment:
Question here , we have machineReadable for EEA but not for GBR . is there assumption all cards are machine readable if it is GBR national ?